internal/core/graphics/image/svg.rs

```diff
@@ -74,6 +77,15 @@ fn with_svg_options<T>(callback: impl FnOnce(&usvg::Options) -> T) -> T {
     callback(&options)
```

Comment:
This function has a 'TODO' a few lines up. `When the font db cache is a feature in corelib, use it:`
Is that not the case now?

Reply:
Ah yes, it is - thanks :)